Client/Frontend/Browser/SearchTelemetry/SearchTelemetry.swift

```diff
@@ -50,4 +50,15 @@ class SearchTelemetry {
     func trackOrganic() {
         GleanMetrics.Search.inContent["\(provider).organic.none"].add()
     }
+    
+    //MARK: Google Top Site SAP
+    func trackGoogleTopSiteTap() {
+        GleanMetrics.Search.googleTopsitePressed["\(SearchEngine.google).\(code)"].add()
+    }
+    
+    func trackGoogleTopSiteFollowOn() {
+        GleanMetrics.Search.inContent["\(SearchEngine.google).in-content.google-topsite-follow-on.\(code)"].add()
```

Comment:
Waiting on this telemetry to be confirmed and then this PR will be good to be opened for review

Reply:
We are still waiting on this particular telemetry to be confirmed in terms of formatting but overall I am opening PR for review.